CMakeLists.txt

```diff
@@ -414,6 +414,7 @@ if (HCC_INTEGRATE_ROCDL)
             -DLLVM_DIR=${CLANG_BIN_DIR}
             -DAMDHSACOD=${AMDHSACOD}
             -DGENERIC_IS_ZERO=ON
+            -DBUILD_HC_LIB=ON
```

Comment:
@dfukalov after checking your earlier comment I think I know the rationale now. I'll hold off this PR and merge it when `ROCm-Device-Libs` does carry this switch.